databricks/koalas/tests/test_groupby.py

```diff
@@ -2608,3 +2608,16 @@ def test_get_group(self):
         self.assertRaises(
             ValueError, lambda: kdf.groupby([("B", "class"), ("A", "name")]).get_group("mammal")
         )
+
+    def test_median(self):
```

Comment:
I'm wondering if a positive test case is needed here? Then we might compare the result DataFrame with a pre-created Koalas DataFrame.

Reply:
Sounds good. I'll add some positive test cases!

Reply:
Thank you!